backend/infrahub/computed_attribute/tasks.py

```diff
@@ -318,195 +307,42 @@ async def trigger_update_jinja2_computed_attributes(
         )
 
 
-@flow(name="computed-attribute-setup", flow_run_name="Setup computed attributes in task-manager")
-async def computed_attribute_setup(
+@flow(name="computed-attribute-setup-jinja2", flow_run_name="Setup computed attributes in task-manager")
+async def computed_attribute_setup_jinja2(
     service: InfrahubServices, context: InfrahubContext, branch_name: str | None = None
 ) -> None:
-    branch_name = branch_name or registry.default_branch
-
-    await add_tags(branches=[branch_name])
-
     log = get_run_logger()
-    await wait_for_schema_to_converge(branch_name=branch_name, service=service, log=log)
 
-    branches_with_diff_from_main = registry.get_altered_schema_branches()
-    schema_branch = registry.schema.get_schema_branch(name=branch_name)
+    if branch_name:
+        await add_tags(branches=[branch_name])
+        await wait_for_schema_to_converge(branch_name=branch_name, service=service, log=log)
```

Comment:
I'm not sure about the indentation here for `wait_for_schema_to_converge` if we end up not sending in branch_name to indicate that it's the default branch. I'll do some tests to validate but if we have anywhere we don't send in the default branch we'd still want to ensure that the schema has converged. Not sure if it's a problem here but highlighting it for later.